diff --git a/dist/cjs/index.d.cts b/dist/cjs/index.d.cts
index 35c5dcfeeb39fed41913098818888e981de695d8..be5678fec78d90969dcd740e9347c849cda908f2 100644
--- a/dist/cjs/index.d.cts
+++ b/dist/cjs/index.d.cts
@@ -1,4 +1,4 @@
 import type { Memoize } from './internalTypes.d.cts';
 export type * from './internalTypes.d.cts';
 export declare const memoize: Memoize;
-export { clearStats, isCollectingStats, getStats, startCollectingStats, stopCollectingStats } from './stats.d.cts';
+export { clearStats, isCollectingStats, getStats, startCollectingStats, stopCollectingStats } from './stats.cjs';
diff --git a/dist/es/index.d.mts b/dist/es/index.d.mts
index 8ee9882779a0967e7d218e7b9343ad8af548e216..0b968b6a9f027330e86891bf099a6b431cf903e7 100644
--- a/dist/es/index.d.mts
+++ b/dist/es/index.d.mts
@@ -1,4 +1,4 @@
 import type { Memoize } from './internalTypes.d.mts';
 export type * from './internalTypes.d.mts';
 export declare const memoize: Memoize;
-export { clearStats, isCollectingStats, getStats, startCollectingStats, stopCollectingStats } from './stats.d.mts';
+export { clearStats, isCollectingStats, getStats, startCollectingStats, stopCollectingStats } from './stats.mjs';
